src/css/queries.scss

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
I want to be able to have all the media queries in a single document but I am unsure on how to do it, I have a couple of ideas, and so far this is working

Reply:
Great work on this. I am also not sure how to put all the media queries into one document. It could be something we could research and work on if we have time.